.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "prettier.configPath": "./config/.prettierrc.json",
+    "eslint.options": {
+        "configFile": "./config/eslintrc.json"
+    }
+}
```

Comment:
You're right, this was an errant file that shouldn't have been added to Git in the first place. I use that file in`shared` since my global setting (pointing to the individual app's `node_modules/pc-nrfconnect-shared/config` of course doesn't work in `shared`.

Reply:
Even though it is good that you removed it here, should we maybe describe how users can configure it in VS Code if they, wish to like https://github.com/NordicSemiconductor/pc-nrfconnect-docs/blob/8f48a0250e71bb73c4b1af4f7eb0608f47e26a4e/app_development.md#using-eslint-in-an-ide?